src/main/java/org/recap/camel/dailyreconciliation/DailyReconciliationProcessor.java

```diff
@@ -162,8 +162,7 @@ public void buildRequestsRows(XSSFWorkbook xssfWorkbook, XSSFSheet xssfSheet, XS
         cellStyle.setAlignment(HorizontalAlignment.LEFT);
         if(StringUtils.isNotBlank(requestId)){
             Optional<RequestItemEntity> requestItemEntity = requestItemDetailsRepository.findById(Integer.valueOf(requestId));
-            if(requestItemEntity != null){
-                ItemEntity itemEntity = requestItemEntity.get().getItemEntity();
+                ItemEntity itemEntity = requestItemEntity.get().getItemEntity();//get().getItemEntity();
```

Comment:
You have removed null check for optional that's good. But you need to check whether it is present or not. So please use isPresent(), before getting data from optional